src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DataStrictnessValidation.feature

```diff
@@ -0,0 +1,352 @@
+/*
+                * SPDX-License-Identifier: Apache-2.0
+                * Licensed to the Ed-Fi Alliance under one or more agreements.
+                * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+                * See the LICENSE and NOTICES files in the project root for more information.
+                */
+
+Feature: Check the least amount of friction in the data exchange while still ensuring data are valid
+
+        Background:
+            Given the system has these "schools" references
+                  | educationOrganizationCategories | gradeLevels   | nameOfInstitution      | schoolId  |
+                  | School                          | Twelfth grade | Grand Bend High School | 255901001 |
+
+        @API-233 @ignore
```

Comment:
Could you confirm if the test is passing, if so could you remove the ignore tag.

Reply:
Fixed